pxr/usdImaging/usdImaging/instanceAdapter.cpp

```diff
@@ -2510,18 +2522,34 @@ UsdImagingInstanceAdapter::GetScenePrimPaths(
         // invisible instances).
         VtIntArray indices = _ComputeInstanceMap(instancerPrim, *instrData, 
             _GetTimeWithOffset(0.0));
-        std::vector<int> remappedIndices;
 
-        remappedIndices.reserve(instanceIndices.size());
-        for (size_t i = 0; i < instanceIndices.size(); i++)
-            remappedIndices.push_back(indices[instanceIndices[i]]);
+		SdfPathVector result;
```

Comment:
Odd indentation here?

Reply:
A few tabs slipped in. I've untabified the file.